### PR TITLE
Updated Denver posting to include dates and location.

### PR DESCRIPTION
**Pull request by tavisto**

### CODE REVIEW

File: site/content/events/2015-denver/_team_global.txt [hunk 1]
```diff
@@ -1,13 +1,5 @@
...
-- John Vincent
-- Andrew Shafer
-- Lindsay Holmwood
-- Stephen Nelson-Smith
-- Christian Trabold
-- James Wickett
-- Anthony Goddard
-- Bernd Erk
-- Bridget Kromhout
+- Tavis Aitken
```
Comment (bridgetkromhout) [inline site/content/events/2015-denver/_team_global.txt]: Please don't remove the global team. Put your team as the local team. :)

**Comment by bridgetkromhout** [comment]
Your local team goes here: https://github.com/jedi4ever/devopsdays-webby/blob/master/site/content/events/2015-denver/_team_local.txt

File: site/content/events/2015-denver/_team_global.txt (hunk 1 above)
Comment (tavisto) [inline site/content/events/2015-denver/_team_global.txt]: Ahh I thought that was a copy paste error. Why is the global team in each event? 
I can fix it just thought it was odd.

**Comment by bridgetkromhout** [comment]
LGTM.

**Comment by bridgetkromhout** [comment]
And @tavisto - I should mention - please put your event on the main-page map too. Here's how Austin's looks:

https://github.com/jedi4ever/devopsdays-webby/blob/master/site/content/index.html#L87-L98

You need a block like that, with your correct lat/long, and you'll have to try different values for the labelAnchor until you find a good one.

Thanks!

**Comment by tavisto** [comment]
Will do, thanks!

File: site/content/events/2015-denver/_team_global.txt (hunk 1 above)
Comment (bridgetkromhout) [inline site/content/events/2015-denver/_team_global.txt]: > Ahh I thought that was a copy paste error. Why is the global team in each event? 
> I can fix it just thought it was odd.

Agreed that it's odd. There are website overhaul things I'd like to do soon that will involve taking the global team out of each event's template; just haven't had time yet. Thanks.

**Pull request merged**